### ISSUE TITLE
Add project(foo) name to CMake

### ISSUE THREAD

**Issue by iBotPeaches** [craftzdog/react-native-quick-md5#15]
```
> Task :react-native-quick-md5:configureCMakeDebug[x86_64]
C/C++: CMake Warning (dev) in CMakeLists.txt:
C/C++:   No project() command is present.  The top-level CMakeLists.txt file must
C/C++:   contain a literal, direct call to the project() command.  Add a line of
C/C++:   code such as
C/C++:     project(ProjectName)
C/C++:   near the top of the file, but after cmake_minimum_required().
C/C++:   CMake is pretending there is a "project(Project)" command on the first
C/C++:   line.
C/C++: This warning is for project developers.  Use -Wno-dev to suppress it.
```

I believe this could be like a simple newline, but unsure the requirement on naming. I'm referencing what I see in vision camera - https://github.com/mrousavy/react-native-vision-camera/blob/main/package/android/CMakeLists.txt#L1

**Comment by craftzdog** [comment]
Let me know if everything is ok, and I'll publish a new version

**Comment by iBotPeaches** [comment]
@craftzdog - Sorry for mega delay. We kinda just used patch-package and forgot about this. It works. Looking to reduce patch-package files so remembered this.

When a 3.0.7 release is cut, we will no longer need a patch file.

**Comment by craftzdog** [comment]
landed in 3.0.7
